LPD-51776 Remove frontend-js-a11y-web and frontend-js-a11y-sample-web

SelenaAungst opened the pull request:
Ticket: https://liferay.atlassian.net/browse/LPD-51776

## Motivation
The A11Y Checker Component is being removed as part of our effort to improve accessibility tooling and simplify our component library. After evaluating its usage and overlap with other accessibility solutions in Liferay, we found it to be redundant. In fact, we stopped using the A11Y Checker some time ago.

## Proposed Solution
Remove frontend-js-a11y-web and frontend-js-a11y-sample-web, as well as all related references.

liferay-continuous-integration commented:
The following guidelines have been set by the owner of this repository: 
- &nbsp;&nbsp;&nbsp;&nbsp;"Thank you for sending this PR to liferay-platform-experience. If you haven't already,take this opportunity to review our [pull request guidelines](https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2915598365). For additional help,please reach out via the [appropriate channels](https://liferay.atlassian.net/wiki/spaces/PEDS/overview#%3Aslack%3A-Slack-Channels)."

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

SelenaAungst commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-29.liferay.com/job/test-portal-source-format/6197/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/6197/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/3256ebbf34d530a6793b547300e9c5d5673fd2cf">3256ebbf34d530a6793b547300e9c5d5673fd2cf</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/SelenaAungst/liferay-portal/tree/LPD-51776-3">LPD-51776-3</a><br/>Branch GIT ID: <a href="https://github.com/SelenaAungst/liferay-portal/commit/f47b86bdf6b9c5bd5afc31f680db6a5f504026b0">f47b86bdf6b9c5bd5afc31f680db6a5f504026b0</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-29.liferay.com/job/test-portal-source-format/6197/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/6197/jenkins-report.html">here</a>.</h5><pre>     [exec] &gt; Task :packageRunCheckFormat
     [exec] yarn run v1.22.22
     [exec] \$ node-scripts check:ci
     [exec] 
     [exec] ⚙️ Running preflight checks...
     [exec] 
     [exec]    · BAD - node-scripts.config.js is out of date, run `yarn run node-scripts generate:global-config`
     [exec] ⚙️ Checking outdated tsconfig.json files ...
     [exec] 
     [exec] ⚙️ Running TypeScript checks on modified files...
     [exec] ℹ️ A total of 12 CPUs were detected: launching tsc using 12 workers
     [exec] 
     [exec] ⚙️ Running format checks on modified files...
     [exec] ❌ CI checks failed.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] &gt; Run with --info or --debug option to get more log output.
     [exec] &gt; Run with --scan to get full insights.
     [exec] &gt; Get more help at https://help.gradle.org.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$1(ExecuteActionsTaskExecuter.java:148)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:282)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:146)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:134)
     [exec]   at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
     [exec]   at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
     [exec]   at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
     [exec]   at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:74)
     [exec]   at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)</pre></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-29.liferay.com/job/test-portal-source-format/6197/">test-portal-source-format#6197</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-29.liferay.com/userContent/jobs/test-portal-source-format/builds/6197/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-platform-experience/liferay-portal/pull/1120">liferay-platform-experience#1120</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/179374400">[master] ci:test:sf - SelenaAungst &gt; liferay-platform-experience - PR#1120 - 2025-03-31[13:43:49]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>179374400</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-29.liferay.com/job/publish-testray-report/24677/">publish-testray-report#24677</a><br/></div></div>

SelenaAungst commented:
ci:test:sf

SelenaAungst commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/af5e046ae69035205f8b9d92f9ab1183b6757b85">af5e046ae69035205f8b9d92f9ab1183b6757b85</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/SelenaAungst/liferay-portal/tree/LPD-51776-3">LPD-51776-3</a><br/>Branch GIT ID: <a href="https://github.com/SelenaAungst/liferay-portal/commit/f47b86bdf6b9c5bd5afc31f680db6a5f504026b0">f47b86bdf6b9c5bd5afc31f680db6a5f504026b0</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-source-format/9986/"><strike>test-portal-source-format</strike></a></li></ul></div><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/9986/jenkins-report.html">here</a>.</h5><pre>     [exec] &gt; Task :packageRunCheckFormat
     [exec] yarn run v1.22.22
     [exec] \$ node-scripts check:ci
     [exec] 
     [exec] ⚙️ Running preflight checks...
     [exec] 
     [exec]    · BAD - node-scripts.config.js is out of date, run `yarn run node-scripts generate:global-config`
     [exec] ⚙️ Checking outdated tsconfig.json files ...
     [exec] 
     [exec] ⚙️ Running TypeScript checks on modified files...
     [exec] ℹ️ A total of 12 CPUs were detected: launching tsc using 12 workers
     [exec] 
     [exec] ⚙️ Running format checks on modified files...
     [exec] ❌ CI checks failed.
     [exec] 
     [exec] 
     [exec] FAILURE: Build failed with an exception.
     [exec] 
     [exec] * What went wrong:
     [exec] Execution failed for task ':packageRunCheckFormat'.
     [exec] &gt; Process 'command '/opt/dev/projects/github/liferay-portal/build/node/bin/node'' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] &gt; Run with --info or --debug option to get more log output.
     [exec] &gt; Run with --scan to get full insights.
     [exec] &gt; Get more help at https://help.gradle.org.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':packageRunCheckFormat'.
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda\$executeIfValid\$1(ExecuteActionsTaskExecuter.java:148)
     [exec]   at org.gradle.internal.Try\$Failure.ifSuccessfulOrElse(Try.java:282)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:146)
     [exec]   at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:134)
     [exec]   at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
     [exec]   at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)
     [exec]   at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)
     [exec]   at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:74)
     [exec]   at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)</pre></details></html>

SelenaAungst commented:
ci:test:sf

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/af5e046ae69035205f8b9d92f9ab1183b6757b85">af5e046ae69035205f8b9d92f9ab1183b6757b85</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/SelenaAungst/liferay-portal/tree/LPD-51776-3">LPD-51776-3</a><br/>Branch GIT ID: <a href="https://github.com/SelenaAungst/liferay-portal/commit/22b8d69e580651e6b409bc5e088ca4b58a9fe089">22b8d69e580651e6b409bc5e088ca4b58a9fe089</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-7.liferay.com/job/test-portal-source-format/6511/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/6511/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-7.liferay.com/job/test-portal-source-format/6511/">test-portal-source-format#6511</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-7.liferay.com/userContent/jobs/test-portal-source-format/builds/6511/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-platform-experience/liferay-portal/pull/1120">liferay-platform-experience#1120</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/180369431">[master] ci:test:sf - SelenaAungst &gt; liferay-platform-experience - PR#1120 - 2025-04-01[14:07:15]</a><br/></div><code><strong>Testray Build ID:</strong><span></span><code>180369431</code><br/></code><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-7.liferay.com/job/publish-testray-report/4399/">publish-testray-report#4399</a><br/></div></div>

thiago-buarqque commented:
@SelenaAungst I still see references to A11Y when I run `git grep -i -C 5 A11Y`. Should they be removed as well?

SelenaAungst commented:
@thiago-buarqque I ended up having to keep some A11y references because it was causing the source formatter to fail, and there were some others that aren't being removed (see https://github.com/liferay-platform-experience/liferay-portal/pull/1106)

thiago-buarqque commented:
@SelenaAungst 
Ah yeah, forgot about Evan's comment on what should not be removed. My bad

About the references that broke SF, which one are those?

SelenaAungst commented:
@thiago-buarqque 
I reverted this commit where I removed some SF changes: https://github.com/liferay-platform-experience/liferay-portal/pull/1106/commits/528154f772b51857f5888e147accc0e45211f7e8

I also reverted this commit of some generated SF changes related to the calendar references I removed: https://github.com/liferay-platform-experience/liferay-portal/pull/1106/commits/cc0b819622ae0ca1e78450c57657b8e6242051e1

SelenaAungst commented:
ci:test:relevant

SelenaAungst commented:
ci:test:platform-experience

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/13947/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13947/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>platform-experience</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/6727/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6727/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:platform-experience - 0 out of 1 jobs passed in 1 minute</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/243db90b8b5d45c7e0ba21e64347cf2b8cd5934e">243db90b8b5d45c7e0ba21e64347cf2b8cd5934e</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/371c8bf5e6778483853849a6361bbb28228cc9b9">371c8bf5e6778483853849a6361bbb28228cc9b9</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/177889732">EE Development Acceptance (master) - 1274 - 2025-03-28[17:01:08]</a></p><details><summary><strong>ci:test:platform-experience - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/6727/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6727/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/6727/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/liferay-platform-experience/liferay-portal/pull/1120">SelenaAungst/LPD-51776-3</a></strong><pre>Unable to rebase master-temp-1743724436508 to master
Rebasing (1/5)
Rebasing (2/5)
error: could not apply f9c0e46c4a632... LPD-51776 Remove frontend-js/frontend-js-a11y-web.</pre></p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/6727/bundles/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a></li></ul>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/243db90b8b5d45c7e0ba21e64347cf2b8cd5934e">243db90b8b5d45c7e0ba21e64347cf2b8cd5934e</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/371c8bf5e6778483853849a6361bbb28228cc9b9">371c8bf5e6778483853849a6361bbb28228cc9b9</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/177889732">EE Development Acceptance (master) - 1274 - 2025-03-28[17:01:08]</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/13947/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13947/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest(master)/13947/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>Please fix <strong>rebase errors</strong> on <strong><a href="https://github.com/liferay-platform-experience/liferay-portal/pull/1120">SelenaAungst/LPD-51776-3</a></strong><pre>Unable to rebase master-temp-1743724421017 to master
Rebasing (1/5)
Rebasing (2/5)
error: could not apply f9c0e46c4a632... LPD-51776 Remove frontend-js/frontend-js-a11y-web.</pre></p></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/13947/bundles/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a></li></ul>

SelenaAungst commented:
resent: https://github.com/liferay-platform-experience/liferay-portal/pull/1129
